lib/floe/workflow/state.rb

```diff
@@ -25,21 +25,35 @@ def initialize(workflow, name, payload)
         @workflow = workflow
         @name     = name
         @payload  = payload
+        # All states must define Next or End except for Choice, Succeed, and Fail
         @end      = !!payload["End"]
+        @next     = payload["Next"]
+        # All states must define a Type
         @type     = payload["Type"]
+        # All states may define a Comment
         @comment  = payload["Comment"]
```

Comment:
These are the only two properties that all states have
![image](https://github.com/ManageIQ/floe/assets/12851112/c59883eb-a017-4b49-959f-37e5aade1d4d)